wlnr/relay_interface/client_rpc.go

```diff
@@ -0,0 +1,82 @@
+package relay_interface
+
+import (
+	"log"
+	"net"
+	"net/rpc"
+	"time"
+)
+
+type ClientRPC struct {
+	callback ClientCallback
+	relay    *rpc.Client
+	listener net.Listener
+}
+
+type ClientRPCMethods struct {
+	client *ClientRPC
+}
+
+func NewClientRPC(callback ClientCallback) Client {
+	client := &ClientRPC{
+		callback: callback,
+	}
+
+	// Open connection to relay server
+	connection, err := net.DialTimeout("tcp", "localhost:7398", time.Duration(10)*time.Second)
+	if err != nil {
+		log.Fatal("Unable to connect to relay server at localhost: ", err)
+		return nil
+	}
+	client.relay = rpc.NewClient(connection)
+	log.Println("Connected to relay server")
+
+	// Open our rpc server
+	rpcLn, err := net.Listen("tcp", ":7399")
+	if err != nil {
+		log.Fatal("Error when listening for RPC calls: ", err)
+		return nil
+	}
+	client.listener = rpcLn
+
+	// Run our rpc server
+	clientMethods := &ClientRPCMethods{
```

Comment:
Blank space before {

Reply:
GoFmt and the rest of our code say it is alright this way. So if you don't complain, I will leave it as it is.

Reply:
Yes, please go ahead. I meant not to have that comment in, because I saw that you ran a formatting tool later. Still getting used to the GitHub interface for reviews ;)